batch/batch/front_end/templates/table_search.html

```diff
@@ -64,4 +64,13 @@
       </form>
     </div>
   </div>
+  <script>
+    document.getElementById("{{ component_name }}-input-box")
+      .addEventListener("keydown", function(e) {
+        if (e.which === 13 && (e.metaKey || e.ctrlKey)) {
```

Comment:
imo this would be a bit easier to read using the value of `e.key` (`"Enter"`) instead of `e.which` (`13`)